internal/errors/resolver/live.go

```diff
@@ -119,16 +108,6 @@ func (*liveErrorResolver) Resolve(err error) (ResolvedResult, bool) {
 		}, true
 	}
 
-	var timeoutError *taskrunner.TimeoutError
-	if errors.As(err, &timeoutError) {
```

Comment:
What is the reason this case is no longer needed?

Reply:
This error has been removed from the underlying library.